### ISSUE TITLE
Rolling Budgie: Flickering of UI when panel is at the Bottom

### ISSUE THREAD

**Issue by Squeezer-CZ** [geckolinux/geckolinux-project#27]
When panel is in the Bottom (which is default config in Geckolinux!) and you click anything on the panel (lets say you open a application menu), it will appear at top at first, but immediately "jump" to bottom. This causes some flickering and it is not nice.

This might be upstream in Budgie, I cannot test it now. Workaround is, surprisingly, move panel to top. It is even nicer that way. :)

**Comment by geckolinux** [comment]
Hi there, thanks for reporting this bug.

I don't like upper panels. But at any rate the flickering is a Budgie issue that I can't control. Please test the newest GeckoLinux Budgie10 release, hopefully released later today.

**Comment by Squeezer-CZ** [comment]
Will do when free time comes around again.
(ie., I will post results later)

**Comment by Squeezer-CZ** [comment]
@geckolinux I have searched Budgie github, I wanted to report it there. I found out that it was reported already and fix should be comming in Budgie 11. 

(Now is 10.2 I think, but problem is that Budgie will be rewritten from Vala to C (that will be version 11) and it will take time, until then, this wont get any fix. Rewriting codebase will take at least a year according to some mention on their github)
